obj.js

```diff
@@ -1,473 +1,575 @@
 import { assert } from "./util.js";
 
```

Comment:
FTR, I wasn't able to review this file yet, because I'm not getting a useful diff, so it's very difficult to review.

Reply:
Were you looking at the "Files changed" as a whole? Try looking at [Changes from **1 commit**](https://github.com/benbucksch/jpc/pull/3/commits/a141078dcc238b187e920f4ae1096885a1f26aec).

Reply:
Thanks, that's useful. I will look akt that.